code/game/machinery/porta_turret/portable_turret.dm

```diff
@@ -171,6 +171,7 @@
 		dat += "Neutralize All Non-Security and Non-Command Personnel: <A href='?src=[REF(src)];operation=shootall'>[stun_all ? "Yes" : "No"]</A><BR>"

```

Comment:
We also already have an option to attack people that are neither security or command

Reply:
> I added the option to shoot people without mindshields in #36226 with the intention that it could be used as a counter to stolen access. With this change that counter is gone as people will just give themselves Captain access, as they already do anyway.

Let me add a "ignore heads" option instead, better?